agent/app/agent.go

```diff
@@ -848,9 +849,8 @@ func (agent *ecsAgent) startAsyncRoutines(
 
 	telemetryMessages := make(chan ecstcs.TelemetryMessage, telemetryChannelDefaultBufferSize)
 	healthMessages := make(chan ecstcs.HealthMessage, telemetryChannelDefaultBufferSize)
-	instanceStatusMessages := make(chan ecstcs.InstanceStatusMessage, telemetryChannelDefaultBufferSize)
```

Comment:
Removing channel here as doctor is not part of the statsEngine. Adding channel here will be unnecessary complexity and coupling.